app/scripts/directives/mobileServiceInstanceRow.js

```diff
@@ -70,6 +72,18 @@
           return 0;
         });
 
+        row.extendedAnnotationsType = row.mobileClient.metadata.annotations ? row.mobileClient.metadata.annotations[extendedAnnotationTypeKey] : undefined;
```

Comment:
Is this to discern what service this is? We also have `spec.externalMetadata.serviceName: ups` available on the service class.

Reply:
Yes, that's the purpose. I actually looked for something like that and couldn't find a key-value like that. Probably searched for wrong keywords.

Gonna delete that annotation in the operator side and in the UI.

thanks for the suggestion